node_modules/global/process.js

```diff
@@ -0,0 +1 @@
+module.exports = require('process');
```

Comment:
@AmitaiB why are there node modules ?

Reply:
@karimMourra I do not know. I just assumed it was a Fastlane or other TE artifact. @mikesalvia ?

Reply:
I don't see how, fastlane has nothing to do with this app, and those node modules are not in the framework directory so they were installed by you in your local project. 
@AmitaiB Please delete all those node modules, did you check your diff ? You have 241 file changes, the majority of which is unnecessary.

Reply:
you even have a `package-lock.json` file in your diff. This is not a javascript project, you shouldn't have that